template_content/{% if ide_vscode %}.vscode{% endif %}/tasks.json

```diff
@@ -59,6 +59,15 @@
         "cwd": "${workspaceFolder}"
       },
       "problemMatcher": []
+    },
+    {
+      "label": "Analyze TEAL contracts with AlgoKit Tealer integration",
+      "command": "algokit",
+      "args": ["task", "analyze", "${workspaceFolder}"],
```

Comment:
Should we run that task against the .algokit directory, as that will contain the template substituted TEAL files, after the tests have been run (debug mode is enabled in the tests).